build.gradle

```diff
@@ -1,8 +1,8 @@
 //file:noinspection GroovyAssignabilityCheck
 //file:noinspection GroovyAccessibility
 plugins {
-    id "architectury-plugin" version "3.4-SNAPSHOT"
-    id "dev.architectury.loom" version "1.0-SNAPSHOT" apply false
+    id "architectury-plugin" version "3.4.146"
+    id "dev.architectury.loom" version "1.2.341" apply false
```

Comment:
`-SNAPSHOT` is recommended, as it will use the lates version.
Like `3.4-SNAPSHOT` and `1.2-SNAPSHOT`